src/ert/gui/model/snapshot.py

```diff
@@ -245,9 +245,7 @@ def _add_partial_snapshot(self, partial: PartialSnapshot, iter_: int):
 
                         for attr in (ids.CURRENT_MEMORY_USAGE, ids.MAX_MEMORY_USAGE):
                             if job.get(ids.DATA) and attr in job.get(ids.DATA):
-                                job_node.data[ids.DATA] = job_node.data[ids.DATA].set(
-                                    attr, job.get(ids.DATA).get(attr)
-                                )
+                                job_node.data[ids.DATA][attr] = job.get(attr)
```

Comment:
This line is never hit in the tests. Maybe add a test? Otherwise just remove the loop.

Reply:
fixed in gui rewrite cf #5850